pytorch_lightning/profiler/pytorch.py

```diff
@@ -69,7 +69,8 @@ def __init__(self, model: nn.Module) -> None:
         self._handles: Dict[str, List["RemovableHandle"]] = {}
 
     def _start_recording_forward(self, _: nn.Module, input: Tensor, record_name: str) -> Tensor:
-        record = record_function(record_name)
+        # Add [pytorch-lightning][module] in name for pytorch profiler to recognize
```

Comment:
Should it be something like `[pl]` instead of `[pytorch-lightning]`? The full name might decrease the readability too much. We could also not have it at all as using the lightning profiler and doing additional profiling without lightning seems really niche.

Reply:
fixed. thanks!